tests/integration/pilot/main_test.go

```diff
@@ -52,6 +67,55 @@ values:
 			}
 			return nil
 		}).
+		Setup(func(ctx resource.Context) error {
```

Comment:
nit: you might consider moving this to a separate named func. it is lengthy enough to justify, i think.